src/cli/README_COAPS.md

```diff
@@ -236,11 +236,26 @@ Sets the content sent by the test resource.
 Done
 ```
 
-### start
+### coaps start \[check-peer-cert\|max-conn-attempts\]
```

Comment:
Should "coaps" be here?  None of the other commands listed use it

Reply:
+1. 

I agree. we should not need the `coaps` here

Reply:
thanks, will fix now